server/tables/tables.go

```diff
@@ -489,8 +489,61 @@ func (ts *TargetStatus) TableName() string {
 	return "TargetStatuses"
 }
 
+// GitHubAppInstallation represents a BuildBuddy GitHub App installation linked
+// to an organization.
+//
+// We'll listen to app uninstallation webhook events to proactively remove these
+// from the DB, but this is not 100% reliable, so GitHub is ultimately the
+// source of truth for whether an installation is valid or not. Typically, the
+// app can optimistically try to create a token for the installation ID, and
+// GitHub will return an error due to the installation no longer existing or no
+// longer being enabled for a particular repo.
+type GitHubAppInstallation struct {
+	Model
+
+	// UserID is the user that registered this installation to BuildBuddy.
+	UserID string `gorm:"not null;default:''"`
+	// GroupID is the group linked to the GitHub app installation.
+	GroupID string `gorm:"primaryKey"`
+	Perms   int
+	// InstallationID is the GitHub app installation ID.
+	InstallationID int64 `gorm:"not null"`
+	// Owner is the GitHub login of the installation (either a user or
+	// organization name). This can be computed from the installation ID, but we
+	// store it here so that we can run queries to associate repos with
+	// installations.
+	Owner string `gorm:"primaryKey"`
```

Comment:
```suggestion
	// UserID is the user that registered this installation to BuildBuddy.
	UserID string `gorm:"not null;default:''"`

	// GroupID is the group linked to the GitHub app installation.
	GroupID string `gorm:"primaryKey"`
	Perms   int

	// InstallationID is the GitHub app installation ID.
	InstallationID int64 `gorm:"not null"`

	// Owner is the GitHub login of the installation (either a user or
	// organization name). This can be computed from the installation ID, but we
	// store it here so that we can run queries to associate repos with
	// installations.
	Owner string `gorm:"primaryKey"`
```